logstash-core/spec/logstash/plugin_spec.rb

```diff
@@ -120,4 +120,33 @@ class LogStash::Filters::Stromae < LogStash::Filters::Base
       end
     end
   end
+
+  describe "subclass initialize" do
```

Comment:
not sure about having this test here as it is, or having an specific test for codecs, filters, outputs, inputs, etc ... I think it would be an awesome addition to have also checking the super call is working as expected.